packages/scripts/README.md

```diff
@@ -10,6 +10,27 @@ See https://6.docs.plone.org/volto/development/i18n.html for more information.
 
 This script is installed in the `node_modules/.bin` directory and can be called via `yarn i18n` or directly in the `scripts` `package.json` part.
 
+## depssync
+
+Given an existing project path, it updates the Volto `dependencies` and `devDependencies` for the existing Volto (`@plone/volto`) version in the `package.json` file.
+
+If you are in the process to upgrade your project to a given Volto version, you should update the `package.json` `@plone/volto` dependencies key to use the updated version, then run the script.
+The script is included in `@plone/scripts` package, you can run it from your project:
+
+```shell
+pnpm run volto-update-deps .
+```
+
+or run it directly from the `.bin` folder.
+
+```shell
+./node_modules/.bin/volto-update-deps.js path/to/my/project
+```
+
+## Deprecated utilites
+
+The following utilities are no longer used by core, nor in projects
```

Comment:
```suggestion
The following utilities are no longer used by core or in projects.
```

Reply:
Is that correct, that _everything_ after this sentence is no longer used?

Reply:
Could be that people still rely on them, so I think it's better left them, they make no harm. 

We can remove them in the next major.

Reply:
In that case, we should make all the subsequent headings children of this heading by adding the required number of `#`. My question was more about the scope of things and how to format them, not whether to keep or remove them.